src/components/home/modals/Send.vue

```diff
@@ -490,6 +499,7 @@ export default {
     },
     created() {
         this.getSuperNodes()
+        this.aliasAddressList = JSON.parse(window.localStorage.getItem(this.defaultAddress)).alias
```

Comment:
Extract `JSON.parse(window.localStorage.getItem(this.defaultAddress)).alias` to a method since it uses multiple times. Also check probably `null` in method.

<img width="622" alt="Screenshot 2020-04-13 at 12 10 30 PM" src="https://user-images.githubusercontent.com/41148850/79091427-cc4ae380-7d7f-11ea-9132-39ccd36e2575.png">